assets/js/app.js

```diff
@@ -51,7 +54,7 @@
                             document.body.className = `holiday ${nextHoliday.class}`
                             document.title = `O Próximo Feriado Nacional é ${nextHoliday.name}`
                             name.innerHTML = nextHoliday.name;
-                            date.innerHTML = nextHoliday.formattedDate;
+                            date.innerHTML = `${nextHoliday.formattedDate}(${nextHoliday.weekDay})`;
```

Comment:
Esqueci de adicionar isso no review, mas também mudaria a formatação, o que acha?

```suggestion
                            date.innerHTML = `${nextHoliday.weekDay}, ${nextHoliday.formattedDate}()`;
```

e aí removeria o `no dia` lá do html

Reply:
Claro, só pra confirmar, gostaria que ficasse:
`quinta-feira, 08/06/2023,`? ou `quinta-feira(DIA),`?
Fiquei confuso porque na sugestão que enviou ficou com `()` vazio no final.

Reply:
Vi que tu já ta resolvendo, então vou dar resolve aqui 🙇